Fluid/Values/FluidValue.cs

```diff
@@ -210,6 +210,16 @@ public static void SetTypeMapping<T>(Func<T, FluidValue> mapping)
                 return mapping;
             }
 
+            foreach (var @interface in type.GetInterfaces())
```

Comment:
I would have expected to use the interface type here. You can see it's not even using the base types.

Reply:
I don't get it.

Reply:
I added this after I tried to add a workaround by adding a custom fluid value, but it did not work either.